apps/bfd-pipeline/bfd-pipeline-rda-grpc-apps/src/main/java/gov/cms/bfd/pipeline/rda/grpc/apps/LoadRdaJsonApp.java

```diff
@@ -193,6 +207,10 @@ private static class Config {
     private Config(ConfigLoader options) {
       hashPepper = options.hexBytes("hash.pepper");
       hashIterations = options.intValue("hash.iterations");
+      dbAuthType =
+          options
+              .enumOption("database.authType", DatabaseOptions.AuthenticationType.class)
+              .orElse(DatabaseOptions.AuthenticationType.JDBC);
       dbUrl = options.stringValue("database.url", "jdbc:hsqldb:mem:LoadRdaJsonApp");
```

Comment:
So we are stiil supporting HSQL?

Reply:
That's default that was already there.  There are still some places in the code base that reference hsqldb (including this).  I believe there is a ticket in the backlog to remove them.